en/02_Developer_Guides/09_Security/08_SQL_Placeholders.md

```diff
@@ -0,0 +1,18 @@
+---
+title: SQL Placeholders
+summary: SQL placeholders in ORM queries
+icon: tachometer-alt
+---
+
+# SQL Placeholders
+
+SQL placeholders are `?` characters used as a placeholder for a value in a SQL query as a way to prevent SQL injection attacks. They are used by default extensively in queries created by the ORM.
+
+Placeholders have been disabled for performance reasons when filtering by an array of integer only values on a column that is either a [DBPrimarykey](api:SilverStripe\ORM\FieldType\DBPrimaryKey) or a [DBForiegnKey](api:SilverStripe\ORM\FieldType\DBForiegnKey). An example of this type of ORM filter is `->filter(['ID' => $ids])` which will turn into a SQL containing `WHERE IN (<ids>)`. 
```

Comment:
Please don't use "placeholders have been disabled" - nothing has been disabled. We're simply not using them for those queries.

Reply:
Updated